ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -206,7 +206,7 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeCompactionCheckFrequencyInHours =
         verifiableProperties.getIntInRange("store.compaction.check.frequency.in.hours", 7 * 24, 1, 365 * 24);
     storeCompactionPolicyFactory = verifiableProperties.getString("store.compaction.policy.factory",
-        "com.github.ambry.store.DefaultCompactionPolicyFactory");
+        "com.github.ambry.store.CompactAllPolicyFactory");
```

Comment:
Are you intentionally making `CompactAll` the default or did you rename this incorrectly?

Reply:
Intentionally - I wanted to change it in #711 but did not change it in code, just documentation.